experimental/packages/opentelemetry-instrumentation-http/test/utils/rawRequest.ts

```diff
@@ -22,7 +22,7 @@ export async function sendRequestTwice(
   port: number
 ): Promise<Buffer> {
   return new Promise((resolve, reject) => {
-    const request = 'GET /raw HTTP/1.1\n\n';
+    const request = `GET /raw HTTP/1.1\r\nHost: ${host}:${port}\r\n\r\n`;
```

Comment:
According to spec `Host` header is required. In node 20 this started to be enforced by default. Spec also requires `/r/n` not `/n`.